cmd/thanos/main.go

```diff
@@ -236,10 +236,6 @@ func registerProfile(mux *http.ServeMux) {
 	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
 	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
 	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
```

Comment:
so those are not added by  `index`?

Reply:
ack, only the profiles accessible via `pprof.Handler(...)` are delegated.